### PR TITLE
#178 improved MockQueryHandlerRunner

### LINKED ISSUE #178: Smaller improvements on MockQueryHandlerRunner
Fixed names of mock objects:
  * Renamed `testing.mock_query_handler_runner.MockQueryHandlerRunner` to `query_handler.python_query_handler_runner.PythonQueryHandlerRunner`
  * Renamed method `PythonQueryHandlerRunner.execute_query()` to `execute_queries()`
  * Renamed `mock_query_result.MockQueryResult` to `python_query_result.PythonQueryResult`

### PR DESCRIPTION
**Pull request by ckunki**
Closes #178 

Fixed names of mock objects:
* Renamed `MockQueryHandlerRunner` to `PythonQueryHandlerRunner`
* Renamed `MockQueryResult` to `PythonQueryResult`

### CODE REVIEW

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py [hunk 1]
```diff
@@ -59,20 +59,20 @@ def run(self) -> ResultType:
     def handle_continue(self, result: Continue) -> Union[Continue, Finish[ResultType]]:
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:59]:
```suggestion
    def _handle_continue(self, result: Continue) -> Union[Continue, Finish[ResultType]]:
```

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py [hunk 2]
```diff
@@ -59,20 +59,20 @@ def run(self) -> ResultType:
...
         self.release_and_create_query_handler_context_of_input_query()
         self.cleanup_query_handler_context()
-        self.execute_query(result.query_list)
+        self.execute_queries(result.query_list)
         input_query_result = self.run_input_query(result)
         result = self._state.query_handler.handle_query_result(input_query_result)
         return result
 
-    def run_input_query(self, result: Continue) -> MockQueryResult:
+    def run_input_query(self, result: Continue) -> PythonQueryResult:
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:67]:
```suggestion
    def _run_input_query(self, result: Continue) -> PythonQueryResult:
```

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py [hunk 3]
```diff
@@ -85,9 +85,9 @@ def handle_finish(self):
     def cleanup_query_handler_context(self):
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:85]:
```suggestion
    def _cleanup_query_handler_context(self):
```

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py [hunk 4]
```diff
@@ -85,9 +85,9 @@ def handle_finish(self):
...
         cleanup_query_list = \
             self._state.top_level_query_handler_context.cleanup_released_object_proxies()
-        self.execute_query(cleanup_query_list)
+        self.execute_queries(cleanup_query_list)
 
-    def execute_query(self, queries: List[Query]):
+    def execute_queries(self, queries: List[Query]):
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:90]:
```suggestion
    def _execute_queries(self, queries: List[Query]):
```

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:1]: all methods except run should be private

File: tests/unit_tests/query_handler_runner/test_python_query_handler_runner.py [hunk 5]
```diff
@@ -199,13 +199,13 @@ def test_continue_finish(temporary_schema_name, top_level_query_handler_context)
...
                 f"""
-                CREATE OR REPLACE VIEW "{temporary_schema_name}"."temp_db_object_2_1" AS\x20
+                CREATE OR REPLACE VIEW "{temporary_schema_name}"."temp_db_object_2_1" AS
                 SELECT 1 as "a";
                 """
             ),
             cleandoc(
                 f"""
-                SELECT\x20
+                SELECT
```
Comment (tkilias) [inline tests/unit_tests/query_handler_runner/test_python_query_handler_runner.py:208]: did you find the space? very good.

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:1]: in run()  the unknown result exception should also contain the value, that makes debugging easier

File: tests/unit_tests/query_handler_runner/test_python_query_handler_runner.py (hunk 5 above)
Comment (ckunki) [reply tests/unit_tests/query_handler_runner/test_python_query_handler_runner.py:208]: :rofl: - I admit it was a failing test forcing me to remove the `\x20`. 
I assumed somebody else applied a change in the meantime ?

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py
Comment (ckunki) [reply exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:1]: See next push.

Comment (ckunki) [reply exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:1]: See next push

File: exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py [hunk 6]
```diff
@@ -101,15 +101,15 @@ def _wrap_return_query(self, input_query: SelectQueryWithColumnDefinition) -> Tu
         temporary_view_name = self._state.input_query_query_handler_context.get_temporary_view_name()
         input_query_create_view_string = cleandoc(
             f"""
-CREATE OR REPLACE VIEW {temporary_view_name.fully_qualified} AS
```
Comment (tkilias) [inline exasol_advanced_analytics_framework/query_handler/python_query_handler_runner.py:104]: here

**Review by tkilias — APPROVED** [review]

**Pull request merged**